server/assets.py

```diff
@@ -47,6 +47,7 @@
 
 student_css = Bundle(
     'css/student.css',
+    'css/highlight.css',
```

Comment:
It seems like including highlight.css with the student bundle causes some alignment issues.  Do you also see this? 

Adding highlight to the common_css bundle seemed to fix this. I think it's fair to include highlight.css in common since it's shared between the staff/student view.

![image](https://cloud.githubusercontent.com/assets/882381/12777202/26837f9c-ca10-11e5-8922-5f279cfd1c50.png)